portfolio/src/main/webapp/script.js

```diff
@@ -105,4 +106,27 @@ function changeNavbarStickiness() {
   } else {
     navbar.classList.remove("sticky");
   }
+}
+
+/*
+ * Fetches the JSON string for comments from the server and display
+ * the comments.
+ */
+function getComments() {
+  fetch("/data").then(response => response.json()).then(comments => {
+    const commentContainer = document.getElementById("comment-container");
+
+    // Build display of comments. 
+    comments.forEach((comment) => {
+      commentContainer.appendChild(createHeadingElement(comment.commenterName + "(" 
+          + comment.commenterEmail + ") says" + "\"" + comment.text +"\""));
```

Comment:
You can use string template 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Done. I added a helper function that uses string template.